### PR TITLE
Arte unlam info

### PR DESCRIPTION
**Pull request by nogueirasjorge**

### CODE REVIEW

File: app/src/main/assets/model.json [hunk 1]
```diff
@@ -495,9 +495,9 @@
...
         {
           "id":"c-arteunlam-5",
-          "name":"Foto 5",
-          "latitude":-34.669859,
-          "longitude":-58.563361,
+          "name":"Pinturas del Siglo XIX",
+          "latitude":-34.669075,
+          "longitude":-58.565078,
           "order":5,
           "description":"",
```
Comment (fedemoglia) [inline app/src/main/assets/model.json]: Is not neccesary the description of this point?

File: app/src/main/assets/model.json [hunk 2]
```diff
@@ -469,11 +469,11 @@
...
-          "name":"Foto 4",
-          "latitude":-34.670215,
-          "longitude":-58.563677,
+          "name":"Arte Siglo XIX",
+          "latitude":-34.669454,
+          "longitude":-58.564937,
           "order":4,
-          "description":"",
+          "description":"Nos dirigimos al último punto de nuestro recorrido, dirigiéndonos hacia las últimas aulas, deteniéndonos a mitad del pabellón.En este sector se exhiben pinturas del siglo 19.\nEntre ellas se destacan la pintura 3 de mayo en Madrid, del pintor aragonés Francisco de Goya, la cual plasma la lucha del pueblo español contra la dominación francesa en el marco del Levantamiento del dos de mayo, al inicio de la Guerra de la Independencia Española. La pintura es oscura, muestra imágenes fuertes y crea el arquetipo del horror en la pintura española.\nOtro cuadro que se destaca es un boceto de Honoré Daumier del cuadro Don Quijote y Sancho Panza. El centro del cuadro está ocupado por Don Quijote, sobre su caballo, muy flacos y estilizado, pareciendo más una caricatura que un dibujo. Sancho Panza, montado en su burro, aparece con los hombros caídos en una posición menos vehemente que su compañero.",
```
Comment (fedemoglia) [inline app/src/main/assets/model.json]: The number of centuries is indicated by roman numbers.

File: app/src/main/assets/model.json (hunk 1 above)
Comment (nogueirasjorge) [inline app/src/main/assets/model.json]: Not really. It's the last point.

File: app/src/main/assets/model.json [hunk 3]
```diff
@@ -469,11 +469,11 @@
...
-          "name":"Foto 4",
-          "latitude":-34.670215,
-          "longitude":-58.563677,
+          "name":"Arte Siglo XIX",
+          "latitude":-34.669454,
+          "longitude":-58.564937,
           "order":4,
-          "description":"",
+          "description":"Nos dirigimos al último punto de nuestro recorrido, dirigiéndonos hacia las últimas aulas, deteniéndonos a mitad del pabellón.En este sector se exhiben pinturas del siglo XIX.\nEntre ellas se destacan la pintura 3 de mayo en Madrid, del pintor aragonés Francisco de Goya, la cual plasma la lucha del pueblo español contra la dominación francesa en el marco del Levantamiento del dos de mayo, al inicio de la Guerra de la Independencia Española. La pintura es oscura, muestra imágenes fuertes y crea el arquetipo del horror en la pintura española.\nOtro cuadro que se destaca es un boceto de Honoré Daumier del cuadro Don Quijote y Sancho Panza. El centro del cuadro está ocupado por Don Quijote, sobre su caballo, muy flacos y estilizado, pareciendo más una caricatura que un dibujo. Sancho Panza, montado en su burro, aparece con los hombros caídos en una posición menos vehemente que su compañero.",
```
Comment (fedemoglia) [inline app/src/main/assets/model.json]: Change *flacos* by *flaco* hahaha!

**Pull request merged**